src/Internal/Serialization/WitnessSet.purs

```diff
@@ -129,9 +129,7 @@ convertRedeemer (T.Redeemer { tag, index, "data": data_, exUnits }) = do
   newRedeemer tag' index' data' exUnits'
 
 convertPlutusDataEffect :: PD.PlutusData -> Effect PDS.PlutusData
```

Comment:
We can just remove this function and use convertPlutusData directly where this function was use. It's exported internally but CTL doesn't use it